cfgmgr/intfmgr.h

```diff
@@ -52,6 +52,7 @@ class IntfMgr : public Orch
     bool isIntfStateOk(const std::string &alias);
     bool isIntfCreated(const std::string &alias);
     bool isIntfChangeVrf(const std::string &alias, const std::string &vrfName);
+    bool isIfaceNameLenOk(const std::string &alias);
```

Comment:
@arfeigin why do you still need this declaration?

Reply:
thanks for the catch, removed